tests/_ast/test_pytest_toplevel.py

```diff
@@ -1,10 +1,11 @@
 # Note that marimo is not repeated in the imports.
 
-import marimo
+# These imports are auto-generated by marimo.
+# Try modifying the source cell definitions,
+# opposed to the following block.
+import pytest
 
-if marimo.import_guard():
-    # This section is auto-generated by marimo.
-    import pytest
+import marimo
```

Comment:
Annoying. Rearranged by ruff.

Adding a conditional or context forces the ordering to work.

```python
import marimo

with marimo.import_notice:
    """These imports are auto-generated by marimo.
     Try modifying the source cell definitions,
     opposed to the following block."""
```

and doesn't seem to cause an error in in mypy ruff or pyright. But seems excessive